[2017-02-23 20:17:00] request.INFO: Matched route "persona". {"route_parameters":{"_controller":"Acuerdos\\GestionBundle\\Controller\\PersonaController::indexAction","_route":"persona"},"request_uri":"http://10.104.190.19/AcuerdosSyst2.0/web/app.php/Gestion/persona/"} []
[2017-02-23 20:17:00] security.DEBUG: Read existing security token from the session. {"key":"_security_default"} []
[2017-02-23 20:17:00] security.DEBUG: User was reloaded from a user provider. {"username":"andy","provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider"} []
[2017-02-23 20:17:00] security.DEBUG: Access denied, the user is neither anonymous, nor remember-me. {"exception":"[object] (Symfony\\Component\\Security\\Core\\Exception\\AccessDeniedException(code: 403): Access Denied. at C:\\wamp\\www\\AcuerdosSyst2.0\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Security\\Http\\Firewall\\AccessListener.php:70)"} []
[2017-02-23 20:17:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException: "Access Denied." at C:\wamp\www\AcuerdosSyst2.0\vendor\symfony\symfony\src\Symfony\Component\Security\Http\Firewall\ExceptionListener.php line 117 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\AccessDeniedHttpException(code: 0): Access Denied. at C:\\wamp\\www\\AcuerdosSyst2.0\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Security\\Http\\Firewall\\ExceptionListener.php:117, Symfony\\Component\\Security\\Core\\Exception\\AccessDeniedException(code: 403): Access Denied. at C:\\wamp\\www\\AcuerdosSyst2.0\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Security\\Http\\Firewall\\AccessListener.php:70)"} []
[2017-02-23 20:17:01] security.DEBUG: Stored the security token in the session. {"key":"_security_default"} []
[2017-02-23 20:17:06] request.INFO: Matched route "persona". {"route_parameters":{"_controller":"Acuerdos\\GestionBundle\\Controller\\PersonaController::indexAction","_route":"persona"},"request_uri":"http://10.104.190.19/AcuerdosSyst2.0/web/app.php/Gestion/persona/"} []
[2017-02-23 20:17:06] security.DEBUG: Read existing security token from the session. {"key":"_security_default"} []
[2017-02-23 20:17:06] security.DEBUG: User was reloaded from a user provider. {"username":"andy","provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider"} []
[2017-02-23 20:17:06] security.DEBUG: Access denied, the user is neither anonymous, nor remember-me. {"exception":"[object] (Symfony\\Component\\Security\\Core\\Exception\\AccessDeniedException(code: 403): Access Denied. at C:\\wamp\\www\\AcuerdosSyst2.0\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Security\\Http\\Firewall\\AccessListener.php:70)"} []
[2017-02-23 20:17:06] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException: "Access Denied." at C:\wamp\www\AcuerdosSyst2.0\vendor\symfony\symfony\src\Symfony\Component\Security\Http\Firewall\ExceptionListener.php line 117 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\AccessDeniedHttpException(code: 0): Access Denied. at C:\\wamp\\www\\AcuerdosSyst2.0\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Security\\Http\\Firewall\\ExceptionListener.php:117, Symfony\\Component\\Security\\Core\\Exception\\AccessDeniedException(code: 403): Access Denied. at C:\\wamp\\www\\AcuerdosSyst2.0\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Security\\Http\\Firewall\\AccessListener.php:70)"} []
[2017-02-23 20:17:06] security.DEBUG: Stored the security token in the session. {"key":"_security_default"} []
[2017-02-23 20:17:43] request.INFO: Matched route "mis_actas". {"route_parameters":{"_controller":"Acuerdos\\FrontendBundle\\Controller\\frontendController::misActasAction","_route":"mis_actas"},"request_uri":"http://10.104.190.19/AcuerdosSyst2.0/web/app.php/Frontend/actas"} []
[2017-02-23 20:17:43] security.DEBUG: Read existing security token from the session. {"key":"_security_default"} []
[2017-02-23 20:17:43] security.DEBUG: User was reloaded from a user provider. {"username":"andy","provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider"} []
[2017-02-23 20:17:43] request.CRITICAL: Uncaught PHP Exception LogicException: "The controller must return a response (null given). Did you forget to add a return statement somewhere in your controller?" at C:\wamp\www\AcuerdosSyst2.0\app\bootstrap.php.cache line 3105 {"exception":"[object] (LogicException(code: 0): The controller must return a response (null given). Did you forget to add a return statement somewhere in your controller? at C:\\wamp\\www\\AcuerdosSyst2.0\\app\\bootstrap.php.cache:3105)"} []
[2017-02-23 20:17:43] security.DEBUG: Stored the security token in the session. {"key":"_security_default"} []
[2017-03-06 20:46:17] request.INFO: Matched route "mis_actas". {"route_parameters":{"_controller":"Acuerdos\\FrontendBundle\\Controller\\frontendController::misActasAction","_route":"mis_actas"},"request_uri":"http://10.104.190.19/AcuerdosSyst2.0/web/app.php/Frontend/actas"} []
[2017-03-06 20:46:17] security.DEBUG: Read existing security token from the session. {"key":"_security_default"} []
[2017-03-06 20:46:17] security.DEBUG: User was reloaded from a user provider. {"username":"rainy","provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider"} []
[2017-03-06 20:46:17] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Validator\Exception\ConstraintDefinitionException: "No default option is configured for constraint Symfony\Component\Validator\Constraints\DateTime" at C:\wamp\www\AcuerdosSyst2.0\vendor\symfony\symfony\src\Symfony\Component\Validator\Constraint.php line 147 {"exception":"[object] (Symfony\\Component\\Validator\\Exception\\ConstraintDefinitionException(code: 0): No default option is configured for constraint Symfony\\Component\\Validator\\Constraints\\DateTime at C:\\wamp\\www\\AcuerdosSyst2.0\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Validator\\Constraint.php:147)"} []
[2017-03-06 20:46:17] security.DEBUG: Stored the security token in the session. {"key":"_security_default"} []
[2017-03-06 20:47:54] request.INFO: Matched route "ajax_person_create". {"route_parameters":{"_controller":"Acuerdos\\GestionBundle\\Controller\\PersonaController::agregarPersonaAction","_route":"ajax_person_create"},"request_uri":"http://10.104.190.19/AcuerdosSyst2.0/web/app.php/Gestion/persona/add/persona"} []
[2017-03-06 20:47:54] security.DEBUG: Read existing security token from the session. {"key":"_security_default"} []
[2017-03-06 20:47:54] security.DEBUG: User was reloaded from a user provider. {"username":"rainy","provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider"} []
[2017-03-06 20:47:54] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\DBALException: "An exception occurred while executing 'INSERT INTO persona (username, password, nombre, cargo, email, is_active, area_id) VALUES (?, ?, ?, ?, ?, ?, ?)' with params ["", "$2y$12$3UXI8nqbZDSJ.xIQk05kU.l36ZgRU87RoEEj4pGAv2uAJT0gecFy2", "", "", "", 1, null]:  SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicata du champ '' pour la clef 'UNIQ_51E5B69BF85E0677'" at C:\wamp\www\AcuerdosSyst2.0\vendor\doctrine\dbal\lib\Doctrine\DBAL\DBALException.php line 91 {"exception":"[object] (Doctrine\\DBAL\\DBALException(code: 0): An exception occurred while executing 'INSERT INTO persona (username, password, nombre, cargo, email, is_active, area_id) VALUES (?, ?, ?, ?, ?, ?, ?)' with params [\"\", \"$2y$12$3UXI8nqbZDSJ.xIQk05kU.l36ZgRU87RoEEj4pGAv2uAJT0gecFy2\", \"\", \"\", \"\", 1, null]:\n\nSQLSTATE[23000]: Integrity constraint violation: 1062 Duplicata du champ '' pour la clef 'UNIQ_51E5B69BF85E0677' at C:\\wamp\\www\\AcuerdosSyst2.0\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\DBALException.php:91, PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicata du champ '' pour la clef 'UNIQ_51E5B69BF85E0677' at C:\\wamp\\www\\AcuerdosSyst2.0\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Statement.php:165)"} []
[2017-03-06 20:47:54] security.DEBUG: Stored the security token in the session. {"key":"_security_default"} []
[2017-03-06 20:48:16] request.INFO: Matched route "ajax_person_create". {"route_parameters":{"_controller":"Acuerdos\\GestionBundle\\Controller\\PersonaController::agregarPersonaAction","_route":"ajax_person_create"},"request_uri":"http://10.104.190.19/AcuerdosSyst2.0/web/app.php/Gestion/persona/add/persona"} []
[2017-03-06 20:48:16] security.DEBUG: Read existing security token from the session. {"key":"_security_default"} []
[2017-03-06 20:48:16] security.DEBUG: User was reloaded from a user provider. {"username":"rainy","provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider"} []
[2017-03-06 20:48:16] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\DBALException: "An exception occurred while executing 'INSERT INTO persona (username, password, nombre, cargo, email, is_active, area_id) VALUES (?, ?, ?, ?, ?, ?, ?)' with params ["", "$2y$12$1zhCPBXUzPIseNCynb42FukwIvUVgCi9b5TQovhRd..MMoKvWOaqe", "", "", "", 1, 2]:  SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicata du champ '' pour la clef 'UNIQ_51E5B69BF85E0677'" at C:\wamp\www\AcuerdosSyst2.0\vendor\doctrine\dbal\lib\Doctrine\DBAL\DBALException.php line 91 {"exception":"[object] (Doctrine\\DBAL\\DBALException(code: 0): An exception occurred while executing 'INSERT INTO persona (username, password, nombre, cargo, email, is_active, area_id) VALUES (?, ?, ?, ?, ?, ?, ?)' with params [\"\", \"$2y$12$1zhCPBXUzPIseNCynb42FukwIvUVgCi9b5TQovhRd..MMoKvWOaqe\", \"\", \"\", \"\", 1, 2]:\n\nSQLSTATE[23000]: Integrity constraint violation: 1062 Duplicata du champ '' pour la clef 'UNIQ_51E5B69BF85E0677' at C:\\wamp\\www\\AcuerdosSyst2.0\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\DBALException.php:91, PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicata du champ '' pour la clef 'UNIQ_51E5B69BF85E0677' at C:\\wamp\\www\\AcuerdosSyst2.0\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Statement.php:165)"} []
[2017-03-06 20:48:16] security.DEBUG: Stored the security token in the session. {"key":"_security_default"} []
